### PR TITLE
Python 3: Port tests in document-policy, dom and import-maps

### PR DESCRIPTION
**Pull request by ziransun**

### CODE REVIEW

**Review by annevk — DISMISSED** [review]

**Review by stephenmcgruer — CHANGES_REQUESTED** [review]
This change appears to regress results, e.g. see https://github.com/web-platform-tests/wpt/pull/24084/checks?check_run_id=757832786

Possibly due to the issues jdm raised.

**Comment by ziransun** [comment]
Couldn't reproduce the test failure locally. Re-triggering the test run. Hope it's okay.

**Comment by stephenmcgruer** [comment]
> Couldn't reproduce the test failure locally. Re-triggering the test run. Hope it's okay.

Definitely ok, but I can trivially reproduce with your patch:

```
./wpt run --log-mach-level=info --log-mach=- --channel=experimental chrome dom/nodes/Document-characterSet-normalization.html
...
 0:12.47 WARNING No module named html
Traceback (most recent call last):
  File "/home/smcgruer/github/wpt/tools/wptserve/wptserve/server.py", line 267, in finish_handling
    handler(request, response)
  File "/home/smcgruer/github/wpt/tools/wptserve/wptserve/handlers.py", line 294, in __call__
    self._set_path_and_load_file(request, response, func)
  File "/home/smcgruer/github/wpt/tools/wptserve/wptserve/handlers.py", line 274, in _set_path_and_load_file
    exec(compile(f.read(), path, 'exec'), environ, environ)
  File "/home/smcgruer/github/wpt/dom/nodes/encoding.py", line 1, in <module>
    from html import escape
ImportError: No module named html
```

I wonder if you have locally installed [this pip module](https://pypi.org/project/html/)? (which isn't part of the standard lib in Python 2)

File: import-maps/common/tools/format_json.py [hunk 1]
```diff
@@ -15,13 +15,13 @@ def main():
...
+                open(filename, u'r'), object_pairs_hook=collections.OrderedDict)
+            with open(filename, u'w') as f:
+                f.write(json.dumps(spec, indent=2, separators=(u',', u': ')))
+                f.write(u'\n')
         except:
             traceback.print_exc()
 
 
-if __name__ == '__main__':
+if __name__ == u'__main__':
```
[the remark this answers is not in the document]
Comment (ziransun) [reply import-maps/common/tools/format_json.py:26]: @Hexcles, there are a few non handlers related python files in other places as well. How do we handle them?

File: dom/nodes/encoding.py [hunk 2]
```diff
@@ -1,5 +1,7 @@
-from cgi import escape
+from html import escape
```
[the remark this answers is not in the document]
Comment (ziransun) [reply dom/nodes/encoding.py]: Code updated. Thank you!

**Comment by ziransun** [comment]
> I wonder if you have locally installed [this pip module](https://pypi.org/project/html/)? (which isn't part of the standard lib in Python 2)

Yes, doesn't look that it's a built-in module for python 2. Code updated.

**Review by stephenmcgruer — APPROVED** [review]

File: import-maps/common/tools/format_json.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (stephenmcgruer) [reply import-maps/common/tools/format_json.py:26]: If it isn't a wptserve handler, they don't need to be updated as part of the wptserve migration. In particular for this case, I believe its only run manually anyway.

[the remark this answers is not in the document]
Comment (ziransun) [reply import-maps/common/tools/format_json.py]: Right. code updated.

**Comment by ziransun** [comment]
Can we merge this in?

**Pull request merged**